Field-Robotics-Lab/dave
# Stratified Current - ROS/Gazebo Plugin Cleanup

No change in functionality - cleans up split between UnderwaterCurrentPlugin and UnderwaterCurrentROSPlugin.   

To verify: 
roslaunch uuv_dave uuv_dave_transient_current_demo.launch 
If the simulation starts normally, the database file was found.

To be extra thorough, if desired:
Change the <database> tag in 
uuv_dave/worlds/uuv_dave_ocean_waves_transientcurrent.world 
from "transientOceanCurrentDatabase.csv" to the full filepath from your home directory, and rerun the simulation.  It should start normally.
Remove the <database> entry from the .world file altogether.  Rerun the simulation.  It should start normally.
